build/presubmit.sh

```diff
@@ -19,6 +19,4 @@ set -x
 
 ./build/check_gofmt.sh .
 ./build/check_boilerplate.sh
-go vet github.com/google/cadvisor/...
-godep go test -v -race -test.short github.com/google/cadvisor/...
-godep go build github.com/google/cadvisor
+make
```

Comment:
As long as you promise to do it :) I'm worried that leaving all these naked `make` calls around is fragile.